compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
I like your wording. But the flag it self should be called `--skip-failed` or something like this.